python/dgl/graphbolt/feature_store.py

```diff
@@ -26,6 +26,22 @@ def read(self, ids: torch.Tensor = None):
         """
         raise NotImplementedError
 
+    def size(self, ids: torch.Tensor = None):
```

Comment:
The size api should not need id as parameter, feature of all ids should have the same size.